### PR TITLE
BC-3743 Moving status alert to nbc login page

### PR DESCRIPTION
**Pull request by Michaellinaresxk**
# Description
- For mobile version we are displaying the modal of status event for 80% of the entire screen.
- Now we have the status triangle displayed on default, brb and nbc homepage when the user is not logged in.

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-3743

## Screenshots of UI changes
default 3743
brb 3743
nbc 3743

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: static/styles/lib/bootstrap/scss/_dropdown.scss [hunk 1]
```diff
@@ -1,3 +1,11 @@
+$breakpoints: (
+  "420px": (width: 291px, max-width: 80vw, margin-left: 69px),
+  "368px": (max-width: 92.4vw, margin-left: 60px),
+  "360px": (max-width: 92.4vw, margin-left: 68px),
+  "350px": (max-width: 92vw, margin-left: 78px),
```
Comment (VikDavydiuk) [inline static/styles/lib/bootstrap/scss/_dropdown.scss:5]: Can we make a responsive design here? And then remove all these breakpoints and not have to add them in the future?

File: static/styles/lib/alerts.scss [hunk 2]
```diff
@@ -163,10 +156,16 @@
 			}
 
 			@include media-breakpoint-down(sm) {
-				left: -290px;
+				left: -324px;
```
Comment (VikDavydiuk) [inline static/styles/lib/alerts.scss:159]: Using negative values is always a bad practice. Please avoid it

File: static/styles/lib/alerts.scss [hunk 3]
```diff
@@ -163,12 +171,14 @@
...
```
Comment (MartinSchuhmacher) [inline static/styles/lib/alerts.scss:174]: I agree, can you find a different solution without negative values? I remember we had that talk in the past, but don't remember the outcome

File: static/styles/lib/header.scss [hunk 4]
```diff
@@ -9,27 +9,7 @@
...
-		@media (max-width: 370px) {
-				width: 70vw;
-				left: -13.5rem !important;
-		}
-
-		@media (max-width: 300px) {
-				width: 70vw;
-				left: -10rem !important;
-			}
+		left: -20.7rem !important;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/header.scss:12]: same here, see above

File: static/styles/lib/alerts.scss [hunk 5]
```diff
@@ -11,18 +11,26 @@
...
 		width: 400px;
 	}
-
-	@media (max-width: 450px) {
-		.dropdown-menu {
-			width: 70vw;
-			margin-left: 80px;
+	@media (min-width: 381px) and (max-width: 420px) {
+		.minimal-button .dropdown-menu {
+			left: -18.2rem!important;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/alerts.scss:16]: 1. same here with negative values
2. might be a bit petty, but please put a blank between the unit and the "!important"

File: static/styles/lib/alerts.scss [hunk 6]
```diff
@@ -11,18 +11,26 @@
...
 		}
+	}
 
-		.dropdown-menu::before,
-		.dropdown-menu::after {
-			content: none !important;
-		}
+@media (min-width: 341px) and (max-width: 380px) {
+	.minimal-button .dropdown-menu {
+    left: -16.9rem!important;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/alerts.scss:23]: see above (both points 1 & 2 from above)

File: static/styles/lib/alerts.scss [hunk 7]
```diff
@@ -11,18 +11,26 @@
...
+    width: 91.5vw;
 	}
+}
+@media (min-width: 320px) and (max-width: 340px) {
+  .minimal-button .dropdown-menu {
+    left: -14.6rem!important;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/alerts.scss:29]: see above (points 1 & 2)

File: static/styles/lib/loggedin.scss [hunk 8]
```diff
@@ -3,6 +3,8 @@
 @import "./editor";
 @import "./alerts.scss";
 
+$transaleModal: translate(50px, 0px);
```
Comment (MartinSchuhmacher) [inline static/styles/lib/loggedin.scss:6]: "transale" is a typo, right?

File: theme/n21/views/lib/header.hbs [hunk 9]
```diff
@@ -6,10 +6,10 @@
                 class="pull-left"
                 href="/"
                 aria-label="{{$t 'global.link.landing'}}"
-            ><div class="cloud-logo"></div></a>
-            {{#if showAlerts}}
+            ><div class="cloud-logo cloud-logo-nbc"></div></a>
+            {{!-- {{#if showAlerts}} --}}
```
Comment (MartinSchuhmacher) [inline theme/n21/views/lib/header.hbs:10]: why keeping comments?

File: theme/n21/views/lib/header.hbs [hunk 10]
```diff
@@ -6,10 +6,10 @@
...
                 {{> "lib/components/alerts"}}
-            {{/if}}
+            {{!-- {{/if}} --}}
```
Comment (MartinSchuhmacher) [inline theme/n21/views/lib/header.hbs:12]: see above

File: views/lib/header.hbs [hunk 11]
```diff
@@ -28,9 +28,9 @@
 			</ul>
 		{{/unless}}
 	</nav>
-	<div class="text-center hidden-sm-up" style="margin-bottom: 10px;">
+	{{!-- <div class="text-center hidden-sm-up" style="margin-bottom: 10px;">
```
Comment (MartinSchuhmacher) [inline views/lib/header.hbs:31]: why keeping comments?

**Review by MartinSchuhmacher — DISMISSED** [review]

**Comment by Michaellinaresxk** [comment]
After carefully analyzing all the implementation of the styles related to the modal, it is impossible not to use these negative values with the code we currently have, since other elements depend on these values.

The negative properties are in the codebase as the only way they present the modal, and I just adjust the values according to the needs of the device.

The only option to avoid these negative values to move the modal would be to rewrite all the code relating to these elements and this action requires another ticket because in the current ticket description it is not related to rewrite or modify files that are not related to my task.

Also this action would be confusing to be able to review my current task, counting that we have the information to dont waste much time on legacy client...   Im trying to implement good practice for sure, but in this case wont work with another solution as flexbox or grid or transform: translate:

**Review by virgilchiriac — APPROVED** [review]

**Review by VikDavydiuk — APPROVED** [review]

**Pull request merged**